myosuite/physics/mj_sim_scene.py

```diff
@@ -1,7 +1,7 @@
 """ =================================================
 Copyright (C) 2018 Vikash Kumar, Copyright (C) 2019 The ROBEL Authors
 Author  :: Vikash Kumar (vikashplus@gmail.com)
-Source  :: https://github.com/vikashplus/robohive
+Source  :: https://github.com/vikashplus/myosuite
```

Comment:
I'd advocate leaving framework links to RoboHive. That's where most of the details are.

Reply:
done